rootfs/bin/backup

```diff
@@ -3,6 +3,8 @@
 export BACKUP_FREQUENCY=${BACKUP_FREQUENCY:-12h}
 export BACKUPS_TO_RETAIN=${BACKUPS_TO_RETAIN:-5}
 
+until psql -l -t >/dev/null 2>&1; do sleep 1; done
```

Comment:
This line should be in "if then - fi"  blocks?

(refs https://github.com/deis/postgres/issues/123#issuecomment-234714776)